numba/analysis.py

```diff
@@ -500,3 +500,44 @@ def rewrite_tuple_len(val, func_ir, called_args):
         print("after".center(80, '*'))
         func_ir.dump()
         print('-' * 80)
+
+
+def find_literal_calls(func_ir, argtypes):
```

Comment:
Just noticed, should this be called `find_literally_calls` ? It's looking to `literally` not `literals`?

Reply:
right, i'll change this